src/adapter/src/coord/timeline.rs

```diff
@@ -216,15 +215,17 @@ impl Coordinator {
             let now_fn = if timeline == &Timeline::EpochMilliseconds {
                 now
             } else {
-                NOW_ZERO.clone()
+                // Timelines that are not `EpochMilliseconds` are initialized to
+                // the smallest possible timestamp.
```

Comment:
I think this comment is only half accurate. The `now_fn` is also used every time we get a write timestamp after the timeline is initialized. I've struggled in the past to explain this, but basically the reason we use the minimum is because the non-`EpochMilliseconds` timelines don't have an external concept of now that all write timestamps need to be advanced to.